__tests__/ExpensiMark-HTML-test.js

```diff
@@ -1402,4 +1402,16 @@ test('Mention', () => {
 
     testString = '@user@DOMAIN.com';
     expect(parser.replace(testString)).toBe('<mention-user>@user@DOMAIN.com</mention-user>');
+
+    testString = '@user@domain.com';
+    expect(parser.replace('<mention-user>@user@domain.com</mention-user>')).toBe(testString);
+
+    testString = '@USER@DOMAIN.COM';
+    expect(parser.replace('<mention-user>@USER@DOMAIN.COM</mention-user>')).toBe(testString);
+
+    testString = '@USER@domain.com';
+    expect(parser.replace('<mention-user>@USER@domain.com</mention-user>')).toBe(testString);
+
+    testString = '@user@DOMAIN.com';
+    expect(parser.replace('<mention-user>@user@DOMAIN.com</mention-user>')).toBe(testString);
```

Comment:
Looks good, but I think we should add these tests to `__tests__/ExpensiMark-HTMLToText-test.js` and `__tests__/ExpensiMark-Markdown-test.js`.

Reply:
Sorry that this is dragging on for a while. Please DM me when you make the updates and we can have faster iterations and get this merged today.